internal/controller/vrg_volrep.go

```diff
@@ -85,6 +91,13 @@ func (v *VRGInstance) reconcileVolRepsAsPrimary() {
 			continue
 		}
 
+		if cg, ok := v.isCGEnabled(pvc); ok {
+			pvcNamespacedName.Name = rmnutil.TrimToK8sResourceNameLength(cg + v.instance.Name)
```

Comment:
The trim here seems to be <=63 chars, but resource names can be upto 256 chars [afaict](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names). Is there an additional limitation here?

Reply:
@ShyamsundarR as VolSync is trimming replicationGroupSourceName and replicationGroupDestinationName to 63 chars, I did the same thing, but maybe it's not relevant for resource name.